nitrogenase_finder.py

```diff
@@ -0,0 +1,54 @@
+"""
+Extension on first project for Olin Software Design Fall 2017
+
+@author: Emma Westerhoff
+
+"""
+
```

Comment:
nice! I like the fact that you've learned the concept of dynamic programming. Something to think about : Now that you've learned about recursion, what would be the pros and cons of using recursion vs dynamic programming to tackle this problem?